draft-lenders-core-coap-dtls-svcb.md

```diff
@@ -102,7 +102,103 @@ same manner as for any other service secured with transport layer security, as
 described in {{-svcb}}. Other authentication mechanisms are currently out of scope.
 
 # Constructing a CoAP URI from a SVCB Resource Record
-TBD Christian
+
+A CoAP client that obtains an SVCB record during the discovery of some service can construct a CoAP request message to interact with.
+Through the rules of {{Section 6.5 of RFC7252}} and {{Section 5.10.2 of RFC7252}}
+(or the equivalent sections for other transports as indicated by the ALPN, eg. {{Section 8.7 of ?RFC8323}}),
+those options can be composed into a URI,
+but contain more information than that.
+
+Requests are initially attempted according to the priorities of the records
+following {{Section 2.4.1 of RFC9460}}.
+Once a request is successful, subsequent requests follow that resolution path
+until the service can not be reached,
+at which point the complete discovery may be reinitiated,
+respecing any rate limits and cachable information along the way.
+
+## Input data
+
+As its input, this algorithm takes a SVCB DNS resource record set (RRs),
+consisting of information about
+
+a. the host name that was queried
+b. the service name that was queried,
+c. whether a / which non-default service port was queried,
+- a set of SVCB records, each containing
+  c. a priority,
+  d. a target name,
+  e. service parameters, and
+  f. additional information about the target (e.g. in the form of AAAA records)
+
+In a DNS request, items a, b and c are grouped using the Attrleaf naming pattern as described in {{Section 2.3 of RFC9460}}.
```

Comment:
```suggestion
In a DNS query, items a, b and c are grouped using the Attrleaf naming pattern as described in {{Section 2.3 of RFC9460}}.
```

Just to avoid confusion between DNS queries and CoAP requests ;-)